easybuild/easyconfigs/n/NAMD/NAMD-2.12-intel-2017a-nompi.eb

```diff
@@ -0,0 +1,29 @@
+name = 'NAMD'
+version = '2.12'
+versionsuffix = '-nompi'
```

Comment:
hmm, isn't `-mt` a better versionsuffix?

also, why no OpenMP support in this one?

Maybe explicitly add `'usempi': False` in `toolchainopts` to highlight the difference?